pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -1,13 +1,17 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, python36Packages }:
```

Comment:
Is this particularly tied to python 3.6? If not, I'd suggest using `python3Packages` here.

Reply:
Agreed.

Reply:
Good idea. Fixed that.